cfg/mujoco_kbot_v1.yml

```diff
@@ -0,0 +1,25 @@
+dt: 0.001  # Simulation timestep in seconds (1/200Hz)
+sim_decimation: 20  # Number of simulation steps per control update (200Hz/4 = 50Hz control)
+gravity: False
+render: True
+joint_id_to_name:
+  11: left_shoulder_pitch_03_ctrl
+  12: left_shoulder_roll_03_ctrl
+  13: left_shoulder_yaw_02_ctrl
+  14: left_elbow_02_ctrl
+  15: left_wrist_02_ctrl
+  21: right_shoulder_pitch_03_ctrl
+  22: right_shoulder_roll_03_ctrl
+  23: right_shoulder_yaw_02_ctrl
+  24: right_elbow_02_ctrl
+  25: right_wrist_02_ctrl
+  31: left_hip_pitch_04_ctrl
+  32: left_hip_roll_03_ctrl
+  33: left_hip_yaw_03_ctrl
+  34: left_knee_04_ctrl
+  35: left_ankle_02_ctrl
+  41: right_hip_pitch_04_ctrl
+  42: right_hip_roll_03_ctrl
+  43: right_hip_yaw_03_ctrl
+  44: right_knee_04_ctrl
+  45: right_ankle_02_ctrl
```

Comment:
btw for config typechecking i strongly prefer using omegaconf: https://omegaconf.readthedocs.io/en/2.3_branch/ good way to provide defaults / check for typos / etc